crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -42,6 +42,9 @@ class SHA3TestVector {
     ASSERT_TRUE(EVP_DigestInit(ctx, algorithm));
     ASSERT_TRUE(EVP_DigestUpdate(ctx, msg_.data(), len_ / 8));
     ASSERT_TRUE(EVP_DigestFinal(ctx, digest, &digest_length));
+    
+    // Alternatively, the correctness can be tested via the single shot API.
```

Comment:
Why do you prefer to leave it commented?

Reply:
I was not sure if we need both API versions to test the functionality (single or combination of APIs) since they overlap. I just added it as a use case or as an example how SHA3 digest can be obtained. However, I can add another separate test based on the single-shot API and also add the `_ex` functions as well if it is a better approach for testing.